### ISSUE TITLE
Release tags

### ISSUE THREAD

**Issue by andvgal** [chrodriguez/redmine_omniauth_saml#36]
Please create release tags, so the plugin looks more mature and suitable for production setups.

For various provisioning systems it would be much easier to work with versions and retrieving implicitly created zips from GitHub, but not live code.

**Comment by chrodriguez** [comment]
You are right. I've created branches for different releases, but it's a must to have tags. I'll take some time to do i.

Thanks
